lib/use_case_pattern/version.rb

```diff
@@ -1,3 +1,3 @@
 module UseCasePattern
-  VERSION = "1.0.2"
+  VERSION = "1.0.4"
```

Comment:
version number does not agree with CHANGELOG.md

Reply:
Thanks, I have updated the changelog and added a step in README